### DISCUSSION TITLE
Issue using lightningcss with Storybook

### DISCUSSION THREAD

**Discussion by smarlhens** [chakra-ui/panda#2140] [Q&A] [answered]
Hey there 👋🏻 

Thanks for your amazing job on this library.

I am trying to use the new `lightningcss: true` option in config but I am facing an issue: it seems that with my current configuration I'm not getting any style in my storybook story: [stackblitz repro repo](https://stackblitz.com/edit/typescript-pandacss-storybook).
Using postcss as mentioned in the documentation https://panda-css.com/docs/installation/storybook it is working well.
Do I need to change something else to make it work ?

As the lightningcss seems to be experimental (not enabled by default) I have decided not to open an issue but a discussion.

Thanks in advance for your help 🙏🏻 

Wish you a great day ☀️ ☮️

**Comment by astahmer** [comment, marked as answer]
the only thing needed to use `lightningcss` is to set `lightningcss: true` in your config, you don't have to update anything else

---
postcss is used in 2 different ways with Panda:
1. internally for some CSS transformations using AST, **this is the part that changes by using `lightningcss: true`**
2. externally as a distribution mechanism (`@pandacss/postcss` or its alias @pandacss/dev/postcss`), as an alternative to the CLI, you can still use `lightningcss: true` with the `@pandacss/dev/postcss` plugin

**Reply by smarlhens** [reply]
Thanks, so I still need to use PostCSS as a distribution mechanism then. I had deleted the `postcss.config.cjs` in my repro repo… I will revert this change. 
Thanks for your answer, wish you a great day 👍🏻
